src/helpers/sender.js

```diff
@@ -8,23 +8,30 @@ dotenv.config();
 const conn = connection(configDB);
 
 exports.sendToFabric = async (username, method, props) => {
+  let response = null;
+  let i = 1;
   const certsList = await DB.getCerts(conn, username);
-  const response = await validator.sendTransaction({
-    identity: {
-      label: username,
-      certificate: certsList[0].cert,
-      privateKey: certsList[0].privatekey,
-      mspId: '482solutions',
-    },
-    network: {
-      channel: 'testchannel',
-      chaincode: 'electricitycc',
-      contract: 'org.fabric.marketcontract',
-    },
-    transaction: {
-      name: method,
-      props,
-    },
-  });
+  while (response === null && i < 10) {
```

Comment:
why we should to send up to 10 transactions instead of just 1?

Reply:
if, for some reason, fabric returns us a write error, that is, the response is null, then we will send the request again

Reply:
why can we just ignore this error?